### PR TITLE
Add troubleshooting documentation

### PR DESCRIPTION
**Pull request by olemarkus**

### CODE REVIEW

File: docs/operations/troubleshoot.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+## Nodeup
+
+Nodeup is the process responsible for the initial provisioning of a node. It is a oneshot systemd service called `kops-configuration.service`. You can see the logs for this service running `journalctl -u kops-configuration.service`.
+
+If it succeed, you should be able to see the following log entries:
+
+```
+nodeup[X]: success
+systemd[1]: kops-configuration.service: Succeeded.
+systemd[1]: Finished Run kops bootstrap (nodeup).
```
Comment (hakman) [inline docs/operations/troubleshoot.md:23]: Would it be a good idea to add a similar entry for Protokube?

Comment (olemarkus) [reply docs/operations/troubleshoot.md:23]: There is a number of things we can add here. Some things around channels could be relevant wrt protokube. but perhaps easier to add as part of helping with a support question on slack. Most of the other content comes from that.

Comment (hakman) [reply docs/operations/troubleshoot.md:23]: OK

**Pull request merged**